src/Adapter/AbstractAdapter.php

```diff
@@ -97,10 +98,25 @@ public function resetFilter($filterName)
     /**
      * {@inheritdoc}
      */
-    public function resetOperationsFilter($filterName)
+    public function resetOperationsFilter($filterName, $value = null)
     {
         if ($this->operationsFilters->offsetExists($filterName)) {
-            $this->operationsFilters->offsetUnset($filterName);
+            if ($value === null) {
+                $this->operationsFilters->offsetUnset($filterName);
+            } else {
```

Comment:
You can remove the else with adding a `continue` after the `offsetUnset` call

Reply:
We can't use `continnue` outside a loop or a switch :thinking: . But I refactor the function :)

Reply:
#fail I see a foreach :p